.vib/osclass/goss/goss.yaml

```diff
@@ -0,0 +1,28 @@
+http:
+  http://osclass:{{ .Vars.service.ports.https }}:
+    status: 200
+    allow-insecure: true
+command:
+  check-user-info:
+    exec: id
+    exit-status: 0
+    stdout:
+      - uid={{ .Vars.containerSecurityContext.runAsUser }}
+      - /groups=.*{{ .Vars.podSecurityContext.fsGroup }}/
+file:
+  /bitnami/osclass/config.php:
+    exists: true
+    filetype: file
+    mode: "0666"
+    contains:
```

Comment:
Could you please also check the correct DB credentials are used? Until now we usually didn't check them explicitly but discovered that the modified values could be parsed incorrectly by the chart and we would know because the app would also work with the default values.

Reply:
As no credentials are used in the configuration file, I'm just checking the database name

Reply:
I think you missed them, but it seems there are entries in the file with the configuration:

<img width="450" alt="Screenshot 2022-09-19 at 09 22 53" src="https://user-images.githubusercontent.com/24253844/190968004-1838dc88-b75c-426f-91b5-777b6592a4cb.png">